modules/@apostrophecms/modal/ui/apos/components/AposDocsManagerToolbar.vue

```diff
@@ -101,23 +91,23 @@ export default {
     },
     labels: {
       type: Object,
-      default () {
-        return {};
-      }
+      default: () => {}
     },
     options: {
       type: Object,
-      default () {
-        return {};
-      }
-    },
-    batchOperations: {
-      type: Array,
-      default: () => []
+      default: () => {}
     },
     displayedItems: {
       type: Number,
       required: true
+    },
+    checkedCount: {
```

Comment:
Some of these required properties are throwing warnings in the media manager modal.

Reply:
I cleaned props and fixed these errors, nicely spotted.